website/project/taxonomies/__init__.py

```diff
@@ -0,0 +1,18 @@
+from modularodm import fields
+
+from framework.mongo import (
+    ObjectId,
+    StoredObject,
+    utils as mongo_utils
+)
+
+
+@mongo_utils.unique_on(['id', '_id'])
```

Comment:
...does `Subject` have a `.id` field? This is a reserved word in python, which is why we use `_id` everywhere